test/Microsoft.TemplateEngine.Cli.UnitTests/ParserTests/TabCompletionTests.cs

```diff
@@ -0,0 +1,263 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.CommandLine;
+using System.CommandLine.Parsing;
+using FakeItEasy;
+using Microsoft.TemplateEngine.Abstractions;
+using Microsoft.TemplateEngine.Cli.Commands;
+using Microsoft.TemplateEngine.Edge;
+using Microsoft.TemplateEngine.Edge.Settings;
+using Microsoft.TemplateEngine.Mocks;
+using Microsoft.TemplateEngine.TestHelper;
+using Xunit;
+
+namespace Microsoft.TemplateEngine.Cli.UnitTests.ParserTests
+{
+    public partial class TabCompletionTests
+    {
+        [Fact]
+        public void Instantiate_CanSuggestTemplateOption_StartsWith()
+        {
+            ITemplateEngineHost host = TestHost.GetVirtualHost(additionalComponents: BuiltInTemplatePackagesProviderFactory.GetComponents(includeTestTemplates: false));
+            var myCommand = NewCommandFactory.Create("new", host, new TelemetryLogger(null, false), new NewCommandCallbacks());
+
+            var parseResult = myCommand.Parse("new console --framework net5.0 --l");
+            var suggestions = parseResult.GetSuggestions().ToArray();
+
+            Assert.Contains("--langVersion", suggestions);
+        }
+
+#pragma warning disable xUnit1004 // Test methods should not be skipped
+        [Fact(Skip = "not valid behavior for parser, should suggest --nuget-source")]
```

Comment:
not expected behavior.
`--interactive` is a boolean `Option`, and parser suggest `True` and `False`, which are not absolutely needed.
Excepted that parser will suggest next option instead (`--add-source`, `--nuget-source`)

Reply:
https://github.com/dotnet/command-line-api/issues/1495

Reply:
Maybe we shouldn't make `--interactive` a boolean. We can just define it as `new Option("--interactive")` instead of `new Option<bool>("--interactive")`.